src/olympia/versions/management/commands/force_min_android_compatibility.py

```diff
@@ -0,0 +1,86 @@
+import csv
+
+from django.core.management.base import BaseCommand
+from django.db.transaction import atomic
+
+import olympia.core.logger
+from olympia import amo
+from olympia.addons.models import Addon
+from olympia.applications.models import AppVersion
+from olympia.versions.models import ApplicationsVersions
+
+
+log = olympia.core.logger.getLogger('z.versions.force_min_android_compatibility')
+
+
+class Command(BaseCommand):
+    """
+    Force current version of add-ons in the specified csv to be compatible with
+    Firefox for Android <MIN_VERSION_FENIX_GENERAL_AVAILABILITY> and higher.
+
+    Should not affect compatibility of add-ons recommended/line for Android.
+    """
+
+    help = (
+        'Force add-ons to be compatible with Firefox for Android '
+        f'{amo.MIN_VERSION_FENIX_GENERAL_AVAILABILITY} and higher'
+    )
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'CSVFILE',
+            help='Path to CSV file containing add-on ids.',
+        )
+
+    def read_csv(self, path):
+        with open(path) as file_:
+            csv_reader = csv.reader(file_)
+            # Format should be a single column with the add-on id.
+            # Ignore non-numeric to avoid the column header.
+            return [int(row[0]) for row in csv_reader if row[0] and row[0].isnumeric()]
```

Comment:
how is this going to handle trailing and leading spaces? (i.e. should we `trim` `row[0]`)?

Reply:
`int()` shouldn't care. I'll add that in the tests

Reply:
`int()` may not care but afaik `isnumeric()` returns false

Reply:
Good point. (`is_numeric()` is also super permissive but it shouldn't be a problem - if wrong data is given we want to blow up anyway...)

Reply:
https://github.com/mozilla/addons-server/pull/21190/commits/536a85359359250fbcae3e6433dc548859bbd0d9